src/App.less

```diff
@@ -22,4 +22,5 @@
   display: flex;
   align-items: center;
   justify-content: center;
+  transform: translateY(125%);
```

Comment:
Is this working as expected for different screen sizes?

Reply:
Yes, I tried it with two different screen sizes.

Reply:
True, it doesn't work correctly when not in full-screen.
See new commit.